proceeding/gammapy-icrc2017.tex

```diff
@@ -404,15 +433,16 @@ \section{Conclusions}
 gamma-ray data analyses.
 
 We find that the Gammapy approach, to build on the powerful and well-tested
-Python packages Numpy and Astropy, brings large benefits: A small codebase that
+Python packages Numpy and Astropy, brings large benefits: a small codebase that
 is focused on gamma-ray astronomy in a single high-level language is easy to
 understand and maintain. It is also easy to modify and extend as new use cases
 arise, which is important for CTA, since it can be expected that the modeling of
 the instrument, background and astrophysical emission, as well as the analysis
 method in general (e.g. likelihood or Bayesian statistical methods) will evolve
 and improve over the next decade. Last but not least, the Gammapy approach is
-inherently collaborative, sharing development effort as well as know-how with
-the larger astronomical community, that to a large degree already has adopted
+inherently collaborative {\bf CB TO BE CHECKED }($~$30 people community of developers){\bf CB }, 
```

Comment:
We have ~ 30 people that have made at least one commit to Gammapy.
I feel that stating that we have a developer community of 30 people is misleading. Really we have ~ 5 people that have contributed a lot.

So I'd suggest to change the phrasing to "30 people have contributed to Gammapy" which can still be misleading, but at least less so. Or state "30 people have contributed at least one commit to Gammapy" which is no longer misleading.

Thoughts?